src/north/console/Console.class.js

```diff
@@ -27,8 +27,12 @@ class Console extends ApiHandler {
   async handleValues(values) {
     if (this.verbose) {
       console.table(values, ['pointId', 'timestamp', 'data'])
+      this.statusData['Last handled Values at'] = new Date().toISOString()
+      this.updateStatusDataStream()
```

Comment:
You can simplify this by updating the status outside the if block (and remove the call in the else block.

Moreover, maybe you can display the last value of the array (content and pointId) like it is done in the protocol connectors

Reply:
Please do so in every north connectors